graph/path/internal/testgraphs/yen_shortest.go

```diff
@@ -0,0 +1,59 @@
+// Copyright ©2018 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package testgraphs
```

Comment:
The tests here are pretty specific to YSK, so having them here is not necessary. I'd just have the test cases in the actual test file.